doc/stages/filters.streamablevoxeldownsize.rst

```diff
@@ -35,3 +36,8 @@ Options
 
 cell
   Cell size in the ``X``, ``Y``, and ``Z`` dimension. [Default: 0.001]
+
+mode
+  Mode for voxel based filtering. [Default: voxelcenter]
+  **voxelcenter**: Point coordinates will be modified and set to the center of a populated voxel. This is a default mode.
```

Comment:
"This is ~a~ **the** default mode." (Or simply omit, as it is stated just above.)